### PR TITLE
[PWN-9468] Pwn 9468 localise

### PR DESCRIPTION
**Pull request by bigearsenal**
## Link jira to issue
https://p2pvalidator.atlassian.net/browse/PWN-9468

## Description of the changes
-

### CODE REVIEW

**Review by zafar4aev — APPROVED** [review]

**Review by TrGiLong — APPROVED** [review]

File: p2p_wallet/AppDelegate/AppDelegateProxyService/IntercomAppDelegateService.swift [hunk 1]
```diff
@@ -4,6 +4,13 @@ import Intercom
 final class IntercomAppDelegateService: NSObject, AppDelegateService {
     // MARK: - Methods
 
+    func application(_: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
+        Intercom.setDeviceToken(deviceToken) { error in
+            guard let error else { return }
+            print("Intercom.setDeviceToken error: ", error)
```
Comment (sidorov-panda) [inline p2p_wallet/AppDelegate/AppDelegateProxyService/IntercomAppDelegateService.swift]: remove?

File: p2p_wallet/AppDelegate/AppDelegateProxyService/LokaliseAppDelegateService.swift [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        #endif
+
+        Lokalise.shared.swizzleMainBundle()
+
+        return true
+    }
+
+    func applicationDidBecomeActive(_: UIApplication) {
+        Lokalise.shared.checkForUpdates { updated, error in
+            if let error { print(error) }
```
Comment (sidorov-panda) [inline p2p_wallet/AppDelegate/AppDelegateProxyService/LokaliseAppDelegateService.swift]: remove?

File: p2p_wallet/Common/Extensions/Bundle+Extensions.swift [hunk 3]
```diff
@@ -40,6 +25,6 @@ extension Bundle {
 
 extension UIApplication {
```
Comment (sidorov-panda) [inline p2p_wallet/Common/Extensions/Bundle+Extensions.swift]: remove extension if not in use?

File: p2p_wallet/AppDelegate/AppDelegateProxyService/IntercomAppDelegateService.swift (hunk 1 above)
Comment (bigearsenal) [reply p2p_wallet/AppDelegate/AppDelegateProxyService/IntercomAppDelegateService.swift]: Ok. I will remove it

File: p2p_wallet/AppDelegate/AppDelegateProxyService/LokaliseAppDelegateService.swift (hunk 2 above)
Comment (bigearsenal) [reply p2p_wallet/AppDelegate/AppDelegateProxyService/LokaliseAppDelegateService.swift]: Removed for production & test. Thanks

File: p2p_wallet/AppDelegate/AppDelegateProxyService/IntercomAppDelegateService.swift (hunk 1 above)
Comment (sidorov-panda) [reply p2p_wallet/AppDelegate/AppDelegateProxyService/IntercomAppDelegateService.swift]: you can put logger here

**Review by sidorov-panda — APPROVED** [review]

**Pull request merged**